openedx/features/pakx/lms/overrides/utils.py

```diff
@@ -55,6 +55,10 @@ def _get_org_log(organization):
     return org_logo.url if org_logo else org_logo
 
 
+def is_blank_str(value):
+    return value.isspace() or not value
```

Comment:
btw `None` should be checked before checking the value.
This function will break if the value is `None`